src/screens/KeyManagement/OCR2KeyCreate.test.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+import { MockedProvider } from '@apollo/client/testing'
+import {
+  OCR2KeysCreate,
+  Props as OCR2KeysCreateProps,
+} from 'screens/KeyManagement/OCR2KeysCreate'
+import { render, screen } from 'test-utils'
+import { OCR2_KEY_FAMILY } from 'hooks/queries/useOCR2KeysQuery'
+import userEvent from '@testing-library/user-event'
+const { queryByText } = screen
+
+function renderComponent(cardProps: OCR2KeysCreateProps, mocks: any[]) {
+  render(
+    <MockedProvider mocks={mocks} addTypename={true}>
+      <OCR2KeysCreate {...cardProps} />
+    </MockedProvider>,
+  )
+}
+
+describe('OCR2KeysCard creation', () => {
+  const familyMocks = {
+    request: {
+      query: OCR2_KEY_FAMILY,
+      variables: {},
+    },
+    result: {
+      data: {
+        __type: {
+          __typename: '__Type',
+          enumValues: [
+            { __typename: '__EnumValue', name: 'EVM' },
+            { __typename: '__EnumValue', name: 'COSMOS' },
+            { __typename: '__EnumValue', name: 'SOLANA' },
+            { __typename: '__EnumValue', name: 'STARKNET' },
+            { __typename: '__EnumValue', name: 'NEW-CHAIN-1' },
+            { __typename: '__EnumValue', name: 'NEW-CHAIN-2' },
+          ],
+        },
+      },
+    },
+  }
+
+  const myObject = {
```

Comment:
```suggestion
  const ocr2KeysCreateProps: OCR2KeysCreateProps = {
```

Reply:
See https://jestjs.io/docs/mock-function-api#jestmockedsource-options for typescript usage